src/test/resources/testenv.yaml

```diff
@@ -1,7 +1,7 @@
 artifactory:
-  protocol: http://
-  url: localhost:8082
-  external_ip: localhost:8082
+  protocol: https://
```

Comment:
localhost and http allowed the test to run in the CI pipeline with no modifications. I'd rather keep it that way. Also, it shows which port is used by RT if it actually runs on the localhost.

Reply:
Sure that is fine.